_site/content/posts/modules/github.md

```diff
@@ -26,7 +26,13 @@ wtf/github/
 ## Required ENV Variables
 
 <span class="caption">Key:</span> `WTF_GITHUB_TOKEN` <br />
-<span class="caption">Action:</span> Your <a href="https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization">Github API</a> token.
+<span class="caption">Action:</span> Your <a href="https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization">Github API</a> token or <a href="https://developer.github.com/enterprise/2.13/v3/enterprise-admin/">Github Enterprise</a> API token.
+
+<span class="caption">Key:</span> `WTF_GITHUB_BASE_URL` <br />
+<span class="caption">Action:</span> Your <a href="https://developer.github.com/enterprise/2.13/v3/enterprise-admin/">Github Enterprise</a> API URL.
+
+<span class="caption">Key:</span> `WTF_GITHUB_UPLOAD_URL` <br />
+<span class="caption">Action:</span> Your <a href="https://developer.github.com/enterprise/2.13/v3/enterprise-admin/">Github Enterprise</a> upload URL (often the same as API URL).
```

Comment:
The documentation needs to be very clear about which ENV vars are required for connecting to GitHub, and which are required for connecting to GitHub Enterprise. I don't think they should be commingled in the docs.

Something like 

```
## GitHub Required ENV Variables

...

## GitHub Enterprise Required ENV Variables

...

```

Reply:
I'm not sure about fixing the docs. Needs more clearance. Maybe setting a new prefix for GE, like `WTF_GE_TOKEN`?

Reply:
My main concern is that there be no confusion by anyone who's not a GitHub Enterprise user about what configuration they need to set up. They're likely to outnumber GHE users by a wide margin.